packages/cli/internal/pkg/cli/clierror/cli_error.go

```diff
@@ -14,8 +15,13 @@ type Error struct {
 }
 
 func (e *Error) Error() string {
-	return fmt.Sprintf("an error occurred invoking '%s'\nwith variables: %+v\ncaused by: %s\nsuggestion: %s",
-		e.Command, e.CommandVars, e.Cause, e.SuggestedAction)
+	if e.SuggestedAction != "" {
```

Comment:
Duplication in string literals can be significantly reduced with a trivial refactoring:

```go
message := fmt.Sprintf("an error occurred invoking '%s'\nwith variables: %+v\ncaused by: %s", e.Command, e.CommandVars, e.Cause)
if e.SuggestedAction != "" {
  message = fmt.Sprintf("%s\nsuggestion: %s", message, e.SuggestedAction)
}
```